README.md

```diff
@@ -671,6 +671,12 @@ const doubledNumbers = numbers.map(doubleN);
 console.log(doubledNumbers); // [0, 2, 4, 6, 8, 10, 12]
 ```
 
+In ES6
```

Comment:
Replace with:

> **Note :** You will frequently encounter this method used in combination with [*arrow functions*](https://github.com/mbeaudru/modern-js-cheatsheet#-arrow-function)
> ```js
> const doubledNumbers = numbers.map(n => n * 2);
> console.log(doubledNumbers); // [0, 2, 4, 6, 8, 10, 12]
> ```

Reply:
I have made the necessary changes as mentioned above